banners/desktop/styles/styles.scss

```diff
@@ -8,6 +8,9 @@
 @use 'src/themes/UseOfFunds/UseOfFunds';
 @use 'src/themes/Treedip/defaults';
 @use 'src/themes/Treedip/ButtonClose/ButtonClose';
+@use 'src/themes/Treedip/ProgressBar/ProgressBar' with (
+	$progress-bar-margin: 0 15px
```

Comment:
what's the reason for this not going into the theme?

Reply:
The iPad used to use the same theme with a different margin IIRC. It can probably be defaulted to `0 15px` in the theme now.